app/models/expression_file_info.rb

```diff
@@ -14,29 +14,43 @@ class ExpressionFileInfo
   UNITS_VALUES = [nil, 'UMI-corrected raw counts', 'raw counts']
   validates :units, inclusion: {in: UNITS_VALUES}
 
-  BIOSAMPLE_INPUT_TYPE_VALUES = [nil, 'whole cell', 'single nuclei', 'bulk']
+  BIOSAMPLE_INPUT_TYPE_VALUES = [nil, 'Whole cell', 'Single nuclei', 'Bulk']
   validates :biosample_input_type, inclusion: {in: BIOSAMPLE_INPUT_TYPE_VALUES}
 
   MULTIMODALITY_VALUES = [nil, 'CITE-seq', 'Patch-seq']
   validates :multimodality, inclusion: {in: MULTIMODALITY_VALUES}
 
   LIBRARY_CONSTRUCTION_VALUES = [nil,
-                                 'Smart-seq2/Fluidigm C1',
-                                 'MARS-seq',
-                                 'Seq-Well v1',
-                                 'Seq-Well S^3',
-                                 'inDrop',
-                                 'sci-RNA-seq',
                                  '10x 3\' v1',
                                  '10x 3\' v2',
                                  '10x 3\' v3',
                                  '10x 5\' v2',
                                  '10x 5\' v3',
                                  'CEL-seq2',
                                  'Drop-seq',
-                                 'SCRB-seq',
-                                 'ATAC-seq',
-                                 'ChIP-seq',
-                                 'methylomics']
+                                 'inDrop',
+                                 'Mars-seq',
```

Comment:
Per similar comment in earlier PR, "MARS" is how it's rendered in literature:

```suggestion
                                 'MARS-seq',
```

Reply:
Eric is correct - many thanks for catching that!